modules/fb_instant_articles_rss/fb_instant_articles_rss.module

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Facebook Instant Articles RSS module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function fb_instant_articles_rss_menu() {
+  $items['fbinstant.rss'] = array(
+    'description' => 'Facebook Instant Articles RSS endpoint',
+    'page callback' => 'fb_instant_articles_rss_page',
+    'file' => 'includes/rss.inc',
+    'access arguments' => array('access content'),
+  );
+
+  $items['admin/config/services/fb-instant-articles/rss'] = array(
+    'title' => 'Facebook Instant Articles RSS',
+    'description' => 'RSS settings for Facebook Instant Articles.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('fb_instant_articles_rss_settings'),
+    'access arguments' => array('administer fb_instant_articles_rss'),
+    'file' => 'includes/admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function fb_instant_articles_rss_permission() {
+  $permissions = array();
+
+  $permissions['administer fb_instant_articles_rss'] = array(
+    'title' => t('Administer Facebook Instant Articles RSS'),
+  );
+  return $permissions;
+}
+
+/**
+ * Implements hook_node_operations().
+ */
+function fb_instant_articles_rss_node_operations() {
```

Comment:
@m4olivei I'm not sure about that. See my logic in https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/17#discussion_r57810546. My assumption is that the Views submodule allows site builders to determine any simple or complex selection criteria they wish for what should display in the Views-controlled feed (whereas the simple RSS submodule really only has one way to do this, hence Views as a more advanced option between these two submodules that do basically the same thing).